examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1932 @@
+{
```

Comment:
<p>mutated? mutilated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Very good point. I think mutation is probably the one to go for. Even though I like the idea of mutilating my DAGS